package.json

```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "csv-stringify": "^1.0.0",
     "jsonwebtoken": "8.2.0",
-    "passport-oauth2": "^1.6.1",
+    "passport-oauth2": "1.4.0",
```

Comment:
This is a regression, we need ^1.6.1 to remain in the package.json. Could the mustache be out of date?